docs/extra-build-config.md

```diff
@@ -389,28 +382,25 @@ option:
         # Raspberry Pi 7\" display/touch screen \n \
         lcd_rotate=2 \n \
         '
-## Enable Raspberry Pi Camera Module
+## Camera Modules
 
-Raspberry Pi does not have the unicam device ( Raspberry Pi Camera ) enabled by default.
-Because this unicam device ( bcm2835-unicam ) as of now is used by libcamera opensource.
-So we have to explicitly enable it in local.conf.
+Automatic camera detection is enabled by default, which is equivalent to:
 
-    RASPBERRYPI_CAMERA_V2 = "1"
+    RASPBERRYPI_CAMERA = "auto"
 
-This will add the device tree overlay imx219 ( Raspberry Pi Camera Module V2 sensor driver 
-) to config.txt. Also, this will enable adding Contiguous Memory Allocation value in the 
-cmdline.txt.
+To override the automatic camera detection, clear `RASPBERRYPI_CAMERA`
+or specify an explicit camera module name. Common values are predefined to
+avoid case sensitivity issues or typos, like `${RASPBERRYPI_CAMERA_V2}`.
+For example, request the support for the Camera Module 3 with either:
 
-Similarly, the Raspberry Pi Camera Module v3 also has to be explicitly enabled in local.conf.
+    RASPBERRYPI_CAMERA = "${RASPBERRYPI_CAMERA_V3}"
```

Comment:
There are more values possible here. I would only keep RASPBERRYPI_CAMERA and have examples of direct assignments instead of having to document indirect variables.